### PR TITLE
Intro to Python part II

### PR DESCRIPTION
**Pull request by SolarDrew**
Creating functions notebook ripped from http://swcarpentry.github.io/python-novice-inflammation/06-func.html.

Further notebooks to follow on this PR

### CODE REVIEW

File: 03-python2-functions-instructors.ipynb [hunk 1]
```diff
@@ -0,0 +1,923 @@
...
+    "- Test and debug a function.\n",
+    "- Set default values for function parameters.\n",
+    "- Explain why we should divide programs into small, single-purpose functions."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "At this point, we’ve written code to draw some interesting features in our inflammation data, loop over all our data files to quickly draw these plots for each of them, and have Python make decisions based on what it sees in our data. But, our code is getting pretty long and complicated; what if we had thousands of datasets, and didn’t want to generate a figure for every single one? Commenting out the figure-drawing code is a nuisance. Also, what if we want to use that code again, on a different dataset or at a different point in our program? Cutting and pasting it is going to make our code get very long and very repetative, very quickly. We’d like a way to package our code so that it is easier to reuse, and Python provides for this by letting us define things called ‘functions’ - a shorthand way of re-executing longer pieces of code.\n",
```
Comment (SolarDrew) [inline 03-python2-functions-instructors.ipynb]: repetative -> repetitive

Comment (dpshelio) [inline 03-python2-functions-instructors.ipynb]: was this a note for you? :-P

File: 03-python2-functions-instructors.ipynb [hunk 2]
```diff
@@ -0,0 +1,923 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "print('freezing point of water:', fahr_to_kelvin(32))\n",
+    "print('boiling point of water:', fahr_to_kelvin(212))"
```
Comment (dpshelio) [inline 03-python2-functions-instructors.ipynb]: I think these two examples would make more sense to me if they are the opposite... I know these temperatures in ºC (or K) but not in ºF, so I would like to convert them from K to F...

File: 03-python2-functions-instructors.ipynb [hunk 3]
```diff
@@ -0,0 +1,923 @@
...
+   "metadata": {},
+   "source": [
+    "### Integer division"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We are using Python 3's print function, where division always returns a floating point number:"
```
Comment (dpshelio) [inline 03-python2-functions-instructors.ipynb]: printing -> division ??

File: 03-python2-functions-instructors.ipynb (hunk 1 above)
Comment (SolarDrew) [inline 03-python2-functions-instructors.ipynb]: Yeah, I noticed it and didn't want to forget :P

**Pull request merged**